charts/camunda-platform-alpha-8.8/values.yaml

```diff
@@ -836,7 +838,15 @@ identityKeycloak:
     ## @param identityKeycloak.image.repository image repo
     repository: camunda/keycloak
     ## @param identityKeycloak.image.tag image tag
-    tag: 25.0.6
+    tag: 26.1.2
+
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 1Gi
+    limits:
+      cpu: 2000m
+      memory: 2Gi
```

Comment:
I imagine these options without the quality comments above each one will flag the CI.

I would suggest `## @skip option.name.resources.limits.cpu` if you do not want our README.md to have those options be viewable in the table linked here:
https://github.com/camunda/camunda-platform-helm/tree/main/charts/camunda-platform-8.6#identity---keycloak-parameters

Reply:
I'm not sure I have an opinion on this, from your perspective is it ok to skip them or should we note them down?

Reply:
I opted to just mirror the modeler note for the same settings